### ISSUE TITLE
Improve final "success" output during CRWA installation (low priority)

### ISSUE THREAD

**Issue by thedavidprice** [redwoodjs/redwood#1171]
<img width="575" alt="Screen Shot 2020-09-16 at 12 33 53 PM" src="https://user-images.githubusercontent.com/2951/93384051-645e6480-f819-11ea-8e80-d41eb7dc5e46.png">

The final output could be better:
- current color always makes me think it's an error
- it's mashed together and hard to parse

Ways to improve:
- change color to something that implies "Success!"
- add some margin (left and bottom) to make block easier to visually scan; would be rad if we could space evenly or maybe put links on line following CTAs with indent (only so much possible on CLI)
- also, we now have a Discord "vanity" URL: https://discord.gg/redwoodjs

Additional ways to improve:
- can we make this more personal/fun --> especially as an invitation to join as a contributor 🤔
- add links to twitter or GH star

**Comment by mojombo** [comment]
Can we also list the Discourse site first while we're at it?

**Comment by jeliasson** [comment]
Work in progress, but something like this @thedavidprice @mojombo?

![image](https://user-images.githubusercontent.com/865493/93854622-5fd0fa80-fcb6-11ea-86a7-b7bd8912c973.png)

**Comment by peterp** [comment]
@jeliasson That's awesome!

**Comment by thedavidprice** [comment]
This is excellent! My choice is Example 2 — the layout and coloring work well and support having three sections.

### Edits
- "Need Help?" --> "Join the Community and Get Help"
- "Join our Discourse Community" --> "Join our Forums"
- "Join our Discord server" --> "Join our Chat"
- "Signup to the Newsletter" --> "Newsletter signup"
- "Want to contribute?" --> "Become a Contributor"
- "Read Introduction" --> "Learn how to get started:"
- "Good starting point: https://..." --> "Find a Good First Issue: https://redwoodjs.com/good-first-issue"

Thoughts and suggestion?

> note: I created a redirect for https://redwoodjs.com/good-first-issue to the GitHub Issues with that label

**Comment by jeliasson** [comment]
> Thoughts and suggestion?

Thanks and good edit suggestions! 👍 
While we get some more feedback, I have made the changes and created a PR with the current implementation (#1195)

**Questions**
* Would you rather have this change in a series of `console.log`'s (as before) or rendered from an [object](https://github.com/redwoodjs/redwood/pull/1195/commits/7373b0de2374383ddd5d906e0bd67bcba14efc92#diff-bb0a9e2525a05a79a2b81b652eb44480R199) (as of PR)?
* Is it okay to do some refactoring of methods and helpers into separate files? If so, I'll will do this in another PR.
* Is there any lightweight core package that would be suitable in the future to share chalk styles/colors for`@redwood/cli` and `@redwood/create-redwood-app`? (See [this](https://github.com/redwoodjs/redwood/blob/main/packages/cli/src/lib/colors.js) and [this](https://github.com/redwoodjs/redwood/pull/1195/commits/7373b0de2374383ddd5d906e0bd67bcba14efc92#diff-bb0a9e2525a05a79a2b81b652eb44480R188))

_Draft 2.1_
![image](https://user-images.githubusercontent.com/865493/93942092-04dbe980-fd30-11ea-97e5-7fedb3af4155.png)

**Comment by thedavidprice** [comment]
@jeliasson Coolio! I'll answer these questions in #1195
